homeassistant/components/homematicip_cloud/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/homematicip_cloud",
   "iot_class": "cloud_push",
   "loggers": ["homematicip"],
-  "requirements": ["homematicip==1.1.5"]
```

Comment:
Please bump in separate PR

Reply:
Done: https://github.com/home-assistant/core/pull/135649